src/main/webapp/app/exam/manage/suspicious-behavior/suspicious-sessions/suspicious-sessions.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { SuspiciousExamSessions, SuspiciousSessionReason } from 'app/entities/exam-session.model';
+import { StudentExam } from 'app/entities/student-exam.model';
+
+@Component({
+    selector: 'jhi-suspicious-sessions',
+    templateUrl: './suspicious-sessions.component.html',
+    styleUrls: ['./suspicious-sessions.component.scss'],
+})
+export class SuspiciousSessionsComponent implements OnInit {
+    @Input() suspiciousSessions: SuspiciousExamSessions;
+    suspiciousFingerprint = false;
+    suspiciousIpAddress = false;
+    suspiciousUserAgent = false;
+    ngOnInit(): void {
+        this.suspiciousFingerprint = this.isSuspiciousFor(SuspiciousSessionReason.SAME_BROWSER_FINGERPRINT);
+        this.suspiciousIpAddress = this.isSuspiciousFor(SuspiciousSessionReason.SAME_IP_ADDRESS);
+        this.suspiciousUserAgent = this.isSuspiciousFor(SuspiciousSessionReason.SAME_USER_AGENT);
+    }
+
+    getStudentExamLink(studentExam: StudentExam) {
+        const studentExamId = studentExam.id;
+        const courseId = studentExam.exam?.course?.id;
+        const examId = studentExam.exam?.id;
+        return `/course-management/${courseId}/exams/${examId}/student-exams/${studentExamId}`;
+    }
+    private isSuspiciousFor = (reason: SuspiciousSessionReason) => {
```

Comment:
```suggestion

    private isSuspiciousFor(reason: SuspiciousSessionReason) {
```

Reply:
Why was this resolved? As far as I see, there was no code change 🤔

Reply:
I added the empty line. Github just does not mark it as outdated.